apps/mobile/src/libs/types/radio-group/radio-group-value.type.ts

```diff
@@ -0,0 +1,6 @@
+type RadioGroupValue = {
```

Comment:
```suggestion
type RadioGroupOption = {
```
I think that would be more clear